CosmicReco/test/Extracted.fcl

```diff
@@ -0,0 +1,71 @@
+#include "Offline/fcl/minimalMessageService.fcl"
```

Comment:
This file (or something very similar) belongs in Production/JobConfig/reco. Then we can add Extracted reco to the validation.

Reply:
This is pretty much the same as JobConfig/reco/Extracted except it produces CosmicTrackDiag ttree instead of art output